content/blog/keda-2.0-alpha.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Give KEDA 2.0 (Alpha) a test drive"
+date = 2020-08-17 # TODO: Update with actual date
+author = "Tom Kerkhove"
++++
+
+Today, we are happy to share that our first **alpha version of KEDA 2.0 is available**! 🎊
+
+# Highlights
+
+With this release, we are shipping majority of our planned features.
+
+Here are some highlights:
+
+- **Making scaling more powerful**
+    - Introduction of `ScaledJob` ([docs](https://keda.sh/docs/2.0/concepts/scaling-jobs/))
+    - Support for scaling Deployments, Stateful Sets and/or any Custom Resources ([docs](https://keda.sh/docs/2.0/concepts/scaling-deployments/))
+    - Support for scaling on standard resource metrics (CPU/Memory)
+    - Support for multiple triggers in a single `ScaledObject` ([docs](https://keda.sh/docs/2.0/concepts/scaling-deployments/))
+    - Support for scaling to original replica count after deleting `ScaledObject` ([docs](https://keda.sh/docs/2.0/concepts/scaling-deployments/))
+    - Support for controling scaling behavior of underlying HPA
+- **Easier to operate KEDA**
+    - Introduction of readiness and liveness probes
+    - Introduction of Prometheus metrics for Metrics Server ([docs](https://keda.sh/docs/2.0/operate/))
+    - Provide more information when quering KEDA resources with `kubectl`
+- **Extensibility**
+    - Introduction of External Push scaler ([docs](https://keda.sh/docs/2.0/scalers/external-push/))
+    - Provide Go client
+
+For a full list of changes, we highly recommend going through [our changelog](https://github.com/kedacore/keda/blob/v2/CHANGELOG.md#v200)!
```

Comment:
I'd add a short note here or somewhere else in the doc, that we will publish a blog with a description of the new features, when 2.0.0 is out.